exts/cesium.omniverse/cesium/omniverse/ui/attributes/data_attributes_widget.py

```diff
@@ -17,8 +17,6 @@ def _customize_props_layout(self, props):
         frame = CustomLayoutFrame(hide_extra=True)
 
         with frame:
-            with CustomLayoutGroup("Access Tokens"):
-                CustomLayoutProperty("cesium:projectDefaultIonAccessToken")
```

Comment:
We can't remove these yet unless we have migration logic. If people created access tokens that they need to copy this needs to be available.

Reply:
It can still be accessed from the Raw USD properties if need be. My plan is to add some migration logic though in a follow up PR.